mmv1/third_party/terraform/utils/node_config.go.erb

```diff
@@ -1033,7 +1030,11 @@ func containerNodePoolLabelsSuppress(k, old, new string, d *schema.ResourceData)
 
 	return true
 }
+<% end -%>
 
+// containerNodePoolTaintSuppress builds a map of taints that represents the diff between
+// the existing and new configuration. Then GKE default taints are removed from the diff
+// and if the resulting diff is empty, the diff is suppressed.
 func containerNodePoolTaintSuppress(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
this function is being added in GA - it seems like it's probably safe because the suppression would still only happen if the changed taints are default / sandbox related.